compiler/luci-interpreter/src/loader/KernelBuilder.cpp

```diff
@@ -31,6 +31,7 @@ namespace luci_interpreter
 enum class BuilderId
 {
 #include <luci/IR/CircleNodes.lst>
+  size // casts to count of values in BuilderId enum
```

Comment:
We can define count of CircleNodes before compilation with this trick =)

Reply:
another alternative is to name it in upper case, so it fits the rest of enum
```suggestion
  SIZE // casts to count of values in BuilderId enum
```

Reply:
sorry, I missed that it uses CLASS instead of OPCODE in the beginning.
changed comment SIZE->Size

Reply:
Done! =)